Testing/SystemTests/tests/framework/ISIS/SANS/WORKFLOWS/SANSSingleReductionTest.py

```diff
@@ -25,6 +27,11 @@
 from sans.user_file.txt_parsers.UserFileReaderAdapter import UserFileReaderAdapter
 
 
+def create_mismatch_name(reference_file_name):
```

Comment:
Perhaps we should move this to a method on the SystemTest class and have that use it for consistency?

Reply:
The tests in this file do not inherit from the SystemTest class as they inherit from the normal UnitTest class, Should I make it a static method on the class and just import it?

Reply:
Yeah I think that would be a good idea to maintain the consistency across all of the tests.